Expensify/App
# [No QA] Upgrade react native CLI to 6.2.0

### Details
Fixes a local build error I'm seeing on Android: https://expensify.slack.com/archives/C01GTK53T8Q/p1641859243262700

**Note:** I followed the custom upgrade instructions [here](https://github.com/react-native-community/cli#updating-the-cli).
**Note:** I had to manually install fbjs https://github.com/wonday/react-native-pdf/pull/592

### Fixed Issues
$ n/a

### Tests
1. Run `rm -rf node_modules`
1. Run `npm i`
1. Run `react-native start --reset-cache`
1. Run the app on all platforms – verify that they build and run successfully.

- [x] Verify that no (new) errors appear in the JS console

### QA Steps
None.

### Tested On

- [ ] ~~Mobile Web~~ This is the same build as web, the client makes no difference wrt to the CLI

### Screenshots
#### Web
![image](https://user-images.githubusercontent.com/47436092/148864079-60ce7bb3-e606-4e23-8679-4bad80a6ac72.png)

#### Desktop
![image](https://user-images.githubusercontent.com/47436092/148864333-14256c14-d3b2-4fcb-95e9-1f3cbf58464c.png)

#### iOS
<img src="https://user-images.githubusercontent.com/47436092/148861547-b14dd6ac-a61d-4712-a5ef-d996b47f6697.png" alt="" width="350px" />

#### Android
<img src="https://user-images.githubusercontent.com/47436092/148862384-3c3c72dc-25c1-4fc1-b22d-6ccfbd65fe4f.png" alt="" width="350px" />
